structuralcodes/materials/constitutive_laws.py

```diff
@@ -461,6 +461,108 @@ def get_ultimate_strain(
         return (100, self._eps_cu1)
 
 
+class Popovics(ConstitutiveLaw):
+    """Class for Popovics-Mander constitutive law.
```

Comment:
Please include a reference to the literature 👍

Reply:
Done